RxJavaTemplate/app/build.gradle

```diff
@@ -52,11 +52,11 @@ android {
     flavorDimensions "stage"
     productFlavors {
         production {
-            applicationId "co.nimblehq"
+            applicationIdSuffix ".production"
```

Comment:
Hmm. Seems the suffix `.production` for production bundle is unnecessary 🤔

Reply:
Yeah, sounds good to me 😁

What do you think @Wadeewee ?

Reply:
Yeah, I agree so we just put it like this 👇  I think it enoung for now 🙏 
`flavorDimensions "stage"
    productFlavors {
        production
        staging
    }
`